src/main/java/by/exadel/internship/controller/FormController.java

```diff
@@ -32,18 +33,21 @@ public FormFullDTO addNewForm(@RequestPart(name = "form") FormRegisterDTO form,
         return formService.process(form, file);
     }
 
+    @PreAuthorize("hasAnyAuthority('SUPER_ADMIN', 'ADMIN')")
```

Comment:
I suggest to introduce 3 custom annotations: RoleAdmin, RoleSuperAdmin and RoleAdminOrSuperAdmin instead of using @PreAuthorize

Reply:
@viktarkolbik @kzaretskaya what do you think about that?

Reply:
It will be more then 3 annotation since there're 3 roles (admin, super-admin, tech-user).
I like idea with custom annotation. How about has 3 annotation:
- @UserAccessControl - - access for role SUPER_ADMIN, ADMIN, TECH_ADMIN
- @AdminAccessControl - - access only for role SUPER_ADMIN, ADMIN
- @SuperAdminAccessControl - access only for role SUPER_ADMIN
 with inheritance between them?

Reply:
Done